samples/azure-iot-hub/main.c

```diff
@@ -12,10 +12,10 @@
 #include "azure_c_shared_utility/platform.h"
 #include "azure_c_shared_utility/threadapi.h"
 #include "azure_c_shared_utility/crt_abstractions.h"
-#include "iothub_client.h"
-#include "iothub_client_options.h"
-#include "iothub_message.h"
-#include "iothubtransportmqtt.h"
+#include "azureiot/iothub_client.h"
```

Comment:
what's the reason that wiringpi.h is being treated as a "standard" lib, and azure iot is not?